### PR TITLE
Recette bugs recepissés

### PR DESCRIPTION
**Pull request by elishowk**
1. Fix CompanySelector.onCompanySelected empty company => empty receipt
2. Fix APPENDIX1 emitter copy to transporter WITH receipt infos
3. Fix TransporterReceipt empty display when a favorite is auto-selected by default
4. Add an `updateForm` test to check that empty transporter company means empty receipt

- [Ticket Favro Annexe 1](https://favro.com/organization/ab14a4f0460a99a9d64d4945/2c84e07578945e0ee8fb61f3?card=tra-12017)
- [Ticket Favro Unselect transporter](https://favro.com/organization/ab14a4f0460a99a9d64d4945/2c84e07578945e0ee8fb61f3?card=tra-11791)

### CODE REVIEW

File: front/src/form/bsdd/Emitter.tsx [hunk 1]
```diff
@@ -59,6 +69,34 @@ export default function Emitter({ disabled }) {
...
+  const { loading, error, refetch } = useQuery<
+    Pick<Query, "companyPrivateInfos">,
+    QueryCompanyPrivateInfosArgs
+  >(COMPANY_SELECTOR_PRIVATE_INFOS, {
+    variables: {
+      clue: orgId!,
+    },
+    skip: !orgId,
+    onCompleted: data => {
+      console.log(data);
```
Comment (JulianaJM) [inline front/src/form/bsdd/Emitter.tsx:84]: console.log

**Review by JulianaJM — APPROVED** [review]

**Review by kornifex — APPROVED** [review]

**Review by benoitguigal — APPROVED** [review]
